.github/workflows/deploy.yml

```diff
@@ -217,15 +220,24 @@ jobs:
         run: |
           terraform apply --auto-approve
           
-      - name: Change Dir
-        id: dir
+      - name: URL Capture
+        id: capture
         run: |
-          cd ../scripts
+          terraform output -raw api_url
 
-      - name: Inject API
+      - name: URL Injection
         id: inject
         run: |
-          bash api-injector.sh
+          chmod +x ../scripts/api-injection.sh
+          ../scripts/api-injection.sh "${{ steps.capture.outputs.stdout }}"
+
+      - name: Push Injection
+        run: |
+          git config --global user.email "billymm314@gmail.com"
+          git config --global user.name "Billy Euceda"
```

Comment:
I like using a special email/user for commits generated by pipelines, I recommend that unless there's a good reason not to. The email doesn't even have to be valid, it can be something like "resume-challenge-back-end@billyeuceda.com"

Reason is, you are not the author of this commit, the pipeline is.

Reply:
okay interesting, didn't think about it that way, will go ahead and change